packages/backend-update/src/auth/router.ts

```diff
@@ -0,0 +1,16 @@
+// import { Router } from 'express'
+// import { AuthController } from './controller'
+
+// export class AuthRouter {
+//   public router!: Router
+
+//   constructor(private authController: AuthController) {
+//     this.router = Router()
+//     this.initializeRoutes()
+//   }
+
+//   private initializeRoutes(): void {
+//     this.router.post('/sign-up', this.authController.register)
+//     this.router.post('/log-in', this.authController.login)
+//   }
+// }
```

Comment:
I don't have a strong opinion against having the routes defined in their own class - my only concern is about having too much abstraction.

What are your opinions? Currently the routes are defined in the `init` method inside `App` class:
https://github.com/interledger/testnet/blob/7a8ffb5542f8fade72bde110360a5849dafd1050/packages/backend-update/src/app.ts#L64-L94

Reply:
Have a separate file to define all the routes can be a solution

Reply:
Having them in a separate file I don't think it's a good solution, since everything lives in a container. 🤔

Reply:
My vote is to keep them in the init method for now. We will figure it out later.